apps/frontend/src/libs/hooks/use-search-filters/use-search-filters.hook.ts

```diff
@@ -20,7 +20,15 @@ const useSearchFilters = ({
 		: "";
 	const [search, setSearch] = useState<string>(searchParameter);
 
+	const [isInitialLoad, setIsInitialLoad] = useState<boolean>(true);
```

Comment:
Are you sure this works as expected? I was thinking maybe the problem in place where we call navigate? like there different methods for navigation, push, replace etc

Reply:
It works only for the redirection from click on contributor item to analytics page. But for example when user is on contributors page and then clicks on sidebar to get to access management page and tries to click back it doesn't go back. It appears to be related to query parameters. I'm unsure if this qualifies as a bug, as the QA team only mentioned that specific redirection in the ticket, so they may consider the current behavior acceptable for these pages.